packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -34,16 +34,42 @@ function isCtrlKeyPressed(e: KeyboardEvent) {
 }
 
 interface SelectableCollectionOptions {
+  /**
+   * An interface for reading and updating multiple selection state.
+   */
   selectionManager: MultipleSelectionManager,
+  /**
+   * A delegate that returns collection item keys with respect to visual layout.
+   */
   keyboardDelegate: KeyboardDelegate,
```

Comment:
Feels like a iffy description, open to suggestions

Reply:
from what I can tell from keyboard delegates like ListKeyboardDelegate and ActionGroupKeyboardDelegate, the delegate gives you getters functions that return keys that are above/left/right/below the key you pass in, not so much updating the layout itself

Reply:
that being said I'm not super happy with the current description but couldn't really come up with an elegant way to say the above

Reply:
```suggestion
  /**
   * A delegate object that implements behavior for keyboard focus movement.
   */
  keyboardDelegate: KeyboardDelegate,
```